pkg/accounting/projectusage_test.go

```diff
@@ -262,3 +253,43 @@ func setUpBucketBandwidthAllocations(ctx *testcontext.Context, projectID uuid.UU
 	}
 	return nil
 }
+
+func TestProjectUsageCustomLimit(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 1, StorageNodeCount: 6, UplinkCount: 1,
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		satDB := planet.Satellites[0].DB
+		acctDB := satDB.ProjectAccounting()
+
+		projectsDB := satDB.Console().Projects()
+		projects, err := projectsDB.GetAll(ctx)
+		require.NoError(t, err)
+
+		project := projects[0]
+		// set custom usage limit for project
+		project.UsageLimit = memory.GiB.Int64() * 10
+		err = projectsDB.Update(ctx, &project)
+		require.NoError(t, err)
+
+		projectUsage := planet.Satellites[0].Accounting.ProjectUsage
+
+		// Setup: create BucketStorageTally records to test exceeding storage project limit
+		now := time.Now()
+		err = setUpStorageTallies(ctx, project.ID, acctDB, now)
```

Comment:
this function `setUpStorageTallies` creates 30GB worth of `BucketStorageTally` records to test the default 25GB project limit.  For this test, I think it would be more meaning to only create 11GB of `BucketStorageTally` records to show that the custom limit is "working"